news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -58,10 +54,10 @@ Instead of assuming a full-alternate playstyle, we assume that **each note of th
 
 The reason for this is to assume the best-case scenario for repeated keystrokes regardless of playstyle. In other words, we aim to approximate the theoretical minimum stamina required to play a given pattern with any playstyle, with any theoretically rollable pattern being rolled. We can't (feasibly), nor is it desirable to, detect playstyles, so we assume the case that requires the least physical repetition, which is to switch keys each time a colour is to be hit, and as such all other existing playstyles will be weaker in comparison to the one we chose.
 
-These changes also help with cases where specific playstyles or patterns were nerfed in the previous rework, as we no longer detect any patterns, stopping at calculating the pure stamina requirement for each map.
-
 ### Readdressing colour
 
+#### Issues within the old system
+
 The old colour system detected the frequency of colour changes and had penalties for specific repeated patterns. This meant that only notes of a different colour than the previous one would receive a non-zero strain, which contributed to issues of complex patterns containing large groups of single-coloured notes not being considered properly in the strain system. Similar to stamina, a penalty was applied to repeated occurrences of single-coloured patterns. While only the 5 most recent occurrences were considered in this penalty, if any two or more of those patterns were repeated, a harsh penalty was applied.
```

Comment:
It's not immediately clear to me what the issues with the old system are (regarding colour).

Reply:
```Previously, the old system detected only the frequency of changes between colours and had specific penalties for particular repeated patterns. This led to complex patterns containing large quantities of single-coloured notes not being considered properly in regards to overall strain values. 

In addition, a penalty was applied to repeat instances of these patterns. This only considered up to five recent occurrences, but if any two or more of such repeats occurred, a harsh penalty was applied.```

is a reword like this more helpful in this regard?